app/(home)/nisp/page.tsx

```diff
@@ -1,10 +1,22 @@
 import PageHero from "@/components/shared/PageHero";
+import Link from "next/link";
 import React from "react";
 
 const Nisp = () => {
   return (
-    <section>
+    <section className="p-3">
       <PageHero pageHeroTitle="NISP" />
+      <div className="max-w-[1500px] mx-auto flex flex-col md:flex-row justify-center items-center gap-8 md:gap-[10%]">
```

Comment:
instead of using flex row , use flex col and place the download button to the rightmost under the description